ledger-core/v2/cmd/insolard/commands.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package main
+
+import (
+	"github.com/insolar/assured-ledger/ledger-core/v2/log/global"
```

Comment:
wrong import order, you need to copy IDE config

Reply:
order correct in this case

Reply:
github.com/insolar/assured-ledger is local, so it should be after external dependency

see example in 
ledger-core/v2/cmd/insolard/main.go

Reply:
goimport said - Nope